docs/pages/enroll-resources/machine-id/reference/configuration.mdx

```diff
@@ -543,6 +543,150 @@ destination:
 - `v1.sock`: the Unix socket that the multiplexer listens on.
 - `agent.sock`: the Unix socket that the SSH agent listens on.
 
+##### Using the SSH multiplexer programmatically
+
+To use the SSH multiplexer programmatically, your SSH client library will need
+to support one of two things:
+
+- The ability to use a ProxyCommand with FDPass. If so, you can use the
+  `ssh_config` file generated by `tbot` to configure the SSH client.
+- The ability to accept an open socket to use as the connection to the SSH
+  server. You will then need to manually connect to the socket and send the
+  multiplexer request.
+
+The `v1.sock` Unix Domain Socket implements the V1 Teleport SSH multiplexer
+protocol. The client must first send a short request message to indicate the
+desired target host and port, terminated will a null byte. The multiplexer will
+then begin to forward traffic to the target host and port. The client can then
+make an SSH connection.
+
+Example in Python using Paramiko:
```

Comment:
Since the code snippets are pretty long relative to each section within this page, I suggest we put each one inside a `Details` box to make the page easier to navigate.

Reply:
Good idea.

Reply:
[53323ea](https://github.com/gravitational/teleport/pull/44676/commits/53323ea827e78f0e3d452d631192b2152b1db8ef)